test/e2e/konnect_test.go

```diff
@@ -68,6 +70,71 @@ func TestKonnectConfigPush(t *testing.T) {
 	requireKonnectNodesConsistentWithK8s(ctx, t, env, rgID, cert, key)
 }
 
+func TestKonnectLicenseActivation(t *testing.T) {
+	t.Parallel()
+	skipIfMissingRequiredKonnectEnvVariables(t)
+
+	ctx, env := setupE2ETest(t)
+
+	rgID := createTestRuntimeGroup(ctx, t)
+	cert, key := createClientCertificate(ctx, t, rgID)
+	createKonnectClientSecretAndConfigMap(ctx, t, env, cert, key, rgID)
+
+	manifestFile := "../../deploy/single/all-in-one-dbless-konnect-enterprise.yaml"
+	t.Logf("deploying %s manifest file", manifestFile)
+
+	manifest := getTestManifest(t, manifestFile)
+	deployKong(ctx, t, env, manifest)
+
+	exposeAdminAPI(ctx, t, env, types.NamespacedName{Namespace: "kong", Name: "proxy-kong"})
+
+	t.Log("disabling license management")
+	kubeconfig := getTemporaryKubeconfig(t, env)
+	cmd := exec.Command("kubectl", "--kubeconfig", kubeconfig, "set", "env", "-n", "kong", "deployment/ingress-kong",
+		"CONTROLLER_KONNECT_LICENSING_ENABLED-")
```

Comment:
```suggestion
		"CONTROLLER_KONNECT_LICENSING_ENABLED=false")
```

Reply:
Left as-is. This setting is false by default; removing it does switch it off.